src/views/workflow-summary-tab/workflow-summary-tab-json-view/workflow-summary-tab-json-view.tsx

```diff
@@ -47,10 +53,24 @@ export default function WorkflowSummaryTabJsonView({
           overrides={overrides.copyButton}
         />
       </div>
-      {activeTab === 'result' && isWorkflowRunning ? (
-        <PrettyJsonSkeleton width="200px" />
-      ) : (
-        <PrettyJson json={jsonMap[activeTab]} />
+      {activeTab === 'input' && <PrettyJson json={jsonMap[activeTab]} />}
+      {activeTab === 'result' && isArchived && (
```

Comment:
I think the if-else logic for the result rendering could be simplified a bit, but I'm not sure if there's a better way to do that besides moving it to a different component.